Source/missiles.cpp

```diff
@@ -1209,17 +1209,40 @@ void InitMissiles()
 
 void AddLArrow(int mi, int sx, int sy, int dx, int dy, int midir, char mienemy, int id, int dam)
 {
+#ifdef HELLFIRE
+	int av;
+
+#endif
 	if (sx == dx && sy == dy) {
 		dx += XDirAdd[midir];
 		dy += YDirAdd[midir];
 	}
 	if (!mienemy) {
+#ifdef HELLFIRE
+		av = 32;

```

Comment:
I think we don't need to have the assignment in the top for Hellfire as it is compiled as C++

```suggestion
		int av = 32;
```